tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
```

Comment:
you're right, @ms-jihua, I'm feeling stupid (again)... it seems that my VS project is a little bit screwed, and I'm not editing the correct file... I'll commit the fix ASAP!